app/services/concerns/xccdf/rule_groups.rb

```diff
@@ -13,31 +13,35 @@ def save_rule_groups
                                            benchmark_id: @benchmark&.id)
         end
 
-        ::RuleGroup.import!(@rule_groups.select(&:new_record?), ignore: true)
+        ::RuleGroup.import!(new_rule_groups, ignore: true)
 
-        ::RuleGroup.import!(rule_group_parents, on_duplicate_key_update: {
-                              conflict_target: %i[ref_id benchmark_id],
-                              columns: %i[description rationale ancestry]
-                            }, validate: false)
+        ::RuleGroup.import(rule_groups_with_ancestry, on_duplicate_key_update: {
```

Comment:
Please add a comment above that here we overwrite a superset of the `old_rule_groups` as the IDs of the ancestors were not available in the first `import!` above.

Reply:
Oh right, I'll add the comments. I wrote down a note about this but missed it 🙈